### PR TITLE
Main

### PR DESCRIPTION
**Pull request by OlegMR0**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```
Comment (okuzan) [inline .gitignore:4]:
```suggestion
logs/*
```

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 2]
```diff
@@ -2,22 +2,27 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login was invoked with '{}' name", login);
```
Comment (okuzan) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:13]:
```suggestion
        logger.debug("Login was invoked with '{}' name", login);
```
"Method was called" type of logs more suitable for debug level

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 3]
```diff
@@ -4,11 +4,15 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder method was invoked with {} id", userId);
```
Comment (okuzan) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:15]:
```suggestion
        logger.debug("completeOrder method was invoked with {} id", userId);
```

File: src/main/resources/log4j2.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <Logger name="mate.academy" level="trace" additivity="false">
+            <AppenderRef ref="LogToFile"/>
+            <AppenderRef ref="LogToConsole"/>
+        </Logger>
+        <Root level="error">
+            <AppenderRef ref="LogToFile"/>
+            <AppenderRef ref="LogToConsole"/>
+        </Root>
+    </Loggers>
+</Configuration>
```
Comment (okuzan) [inline src/main/resources/log4j2.xml:28]: add new line

**Review by okuzan — CHANGES_REQUESTED** [review]
Good job overall!

**Review by olekskov — APPROVED** [review]
Good job

**Pull request closed without merging**